compiler/rustc_trait_selection/src/traits/project.rs

```diff
@@ -213,7 +213,18 @@ fn project_and_unify_type<'cx, 'tcx>(
 
     let infcx = selcx.infcx();
     // FIXME(associated_const_equality): Handle consts here as well as types.
-    let obligation_pred_ty = obligation.predicate.term.ty().unwrap();
+    // FIXME(compiler-errors): We need to normalize here until we properly handle
+    // GAT projection types that we can't normalize properly, see note in
+    // `rustc_trait_selection::traits::project`
```

Comment:
This...is the note?

Reply:
lmao sorry, copied this from another file and it's definitely ambiguous now.

I meant the note about trying to normalize GAT projections then treating them differently if they fail to normalize (not inserting a type variable because it would capture escaping bounds). I think I linked it in the PR description. I'll fix this comment when I get to my computer.